src/pages/resources.tsx

```diff
@@ -114,16 +113,6 @@ const Resources: FunctionComponent = () => {
                 background="mars"
                 title="Get started with Sourcegraph"
                 description="Want to see Sourcegraph in action? Schedule time with a Sourcegraph expert or try Sourcegraph for free."
```

Comment:
We've got a dangler. We may have to adjust the width of the CtaSection, or add a line break `\n`. Not a huge fan of line breaks though since content doesn't flow naturally when wrapping on smaller viewports.

![Screen Shot 2022-09-23 at 12 13 28 AM](https://user-images.githubusercontent.com/1733936/191890905-00692153-340d-4ee0-a39f-64e80b52ca70.png)

Reply:
A "dangler" 😆  -- Added a max width that only applies on md & up screens! Tested on all the centered Cta's, looks good from my end